code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -872,7 +872,7 @@
 	name = "Nitric acid"
 	description = "Nitric acid is an extremely corrosive chemical substance that violently reacts with living organic tissue."
 	color = "#5050FF"
-	toxpwr = 2
+	toxpwr = 3
 	acidpwr = 5.0
```

Comment:
why is acidpwr so low? is this intended or missclick?

Reply:
it was always low